### PR TITLE
feat(tokens): Remove JS tokens

### LINKED ISSUE #1016: Remove javascript tokens
We want to streamline use of our tokens to only provide CSS variables (and classes in the future).

We therefore want to remove tokens in js format due to several factors:

- Lack of support for css functions (`clamp`, `calc` etc.) in js
- Maintenance overhead for supporting js tokens
- js tokens are not used much

### PR DESCRIPTION
**Pull request by Barsnes**
Resolves #1016

### CODE REVIEW

**Review by mimarz — CHANGES_REQUESTED** [review]
Looks good to be mostly good. You forgot to update/remove the `README.MD` in regards to javascript tokens.

File: packages/tokens/scripts/build.ts [hunk 1]
```diff
@@ -173,17 +135,6 @@ const getStorefrontConfig = (brand: Brands, targetFolder = ''): Config => {
...
-        prefix,
-        basePxFontSize,
-        transforms: ['ts/resolveMath', nameKebab.name],
-        files: [
-          {
-            format: 'global-values-hack',
-            destination: 'ignore/hack',
-          },
-        ],
-      },
```
[the remark this answers is not in the document]
Comment (Barsnes) [reply packages/tokens/scripts/build.ts:186]: Gotcha, will add back 👍🏻

**Comment by mimarz** [comment]
This change also has implications for our end users so prefix should be feat(tokens) so we get a version bump and changelog entry :)

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**